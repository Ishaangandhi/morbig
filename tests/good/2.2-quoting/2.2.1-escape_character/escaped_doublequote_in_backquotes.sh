x="`sed -e \"\\\\'etc 'p\" /dev/null`"
