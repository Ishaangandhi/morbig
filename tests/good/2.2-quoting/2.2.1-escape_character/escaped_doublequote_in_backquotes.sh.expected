[
  [
    "CompleteCommand_CList",
    [
      "CList_AndOr",
      [
        "AndOr_Pipeline",
        [
          "Pipeline_PipeSequence",
          [
            "PipeSequence_Command",
            [
              "Command_SimpleCommand",
              [
                "SimpleCommand_CmdPrefix",
                [
                  "CmdPrefix_AssignmentWord",
                  [
                    [ "Name", "x" ],
                    [
                      "Word",
                      "\" ` echo \\\"  \\\\\\\\' \\\"  ` \"",
                      [
                        [
                          "WordDoubleQuoted",
                          [
                            "Word",
                            " ` echo \\\"  \\\\\\\\' \\\"  ` ",
                            [
                              [ "WordLiteral", " " ],
                              [
                                "WordSubshell",
                                [ "SubShellKindBackQuote" ],
                                [
                                  [
                                    "CompleteCommand_CList",
                                    [
                                      "CList_AndOr",
                                      [
                                        "AndOr_Pipeline",
                                        [
                                          "Pipeline_PipeSequence",
                                          [
                                            "PipeSequence_Command",
                                            [
                                              "Command_SimpleCommand",
                                              [
                                                "SimpleCommand_CmdName_CmdSuffix",
                                                [
                                                  "CmdName_Word",
                                                  [
                                                    "Word",
                                                    "echo",
                                                    [
                                                      [ "WordName", "echo" ]
                                                    ]
                                                  ]
                                                ],
                                                [
                                                  "CmdSuffix_Word",
                                                  [
                                                    "Word",
                                                    "\\\"  \\\\\\\\' \\\"",
                                                    [
                                                      [ "WordLiteral", "\\" ],
                                                      [
                                                        "WordDoubleQuoted",
                                                        [
                                                          "Word",
                                                          "  \\\\\\\\' \\",
                                                          [
                                                            [
                                                              "WordLiteral",
                                                              "  \\\\\\\\' \\"
                                                            ]
                                                          ]
                                                        ]
                                                      ]
                                                    ]
                                                  ]
                                                ]
                                              ]
                                            ]
                                          ]
                                        ]
                                      ]
                                    ]
                                  ]
                                ]
                              ],
                              [ "WordLiteral", " " ]
                            ]
                          ]
                        ]
                      ]
                    ]
                  ]
                ]
              ]
            ]
          ]
        ]
      ]
    ]
  ]
]